### PR TITLE
[5.2] Specify Authentication guard on AuthenticatesUsers trait

### PR DESCRIPTION
**Pull request by hiendv**

### CODE REVIEW

File: src/Illuminate/Foundation/Auth/AuthenticatesUsers.php [hunk 1]
```diff
@@ -162,6 +164,15 @@ public function loginUsername()
     }
 
     /**
+     * Get the guard to be used by Auth.
+     * @return string|null
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Auth/AuthenticatesUsers.php]: missing newline before this annotation

File: src/Illuminate/Foundation/Auth/AuthenticatesUsers.php [hunk 2]
```diff
@@ -169,7 +180,8 @@ public function loginUsername()
     protected function isUsingThrottlesLoginsTrait()
     {
         return in_array(
-            ThrottlesLogins::class, class_uses_recursive(get_class($this))
+            ThrottlesLogins::class,
+            class_uses_recursive(get_class($this))
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Auth/AuthenticatesUsers.php]: Please don't change this unrelated code.

**Comment by hiendv** [comment]
@GrahamCampbell Sorry. I've already updated the PR.

File: src/Illuminate/Foundation/Auth/AuthenticatesUsers.php [hunk 3]
```diff
@@ -162,6 +164,16 @@ public function loginUsername()
     }
 
     /**
+     * Get the guard to be used by Auth.
+     *
+     * @return string|null
+     */
+    public function getGuard()
```
Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Auth/AuthenticatesUsers.php]: This shouldn't be public.

Comment (GrahamCampbell) [inline src/Illuminate/Foundation/Auth/AuthenticatesUsers.php]: I think, right?

Comment (hiendv) [inline src/Illuminate/Foundation/Auth/AuthenticatesUsers.php]: Hmm, I didn't think much about the visibility of the method. There is no difference between these getters: `getGuard`, `loginUsername`, `redirectTo`, ..vv right?
